content/docs/faq.md

```diff
@@ -210,3 +217,59 @@ Our deployment is running in [OpenShift
 Online](https://www.openshift.com/products/online/) which is using docker as a
 container engine, that's why we are using docker here and not
 [podman](https://github.com/containers/podman).
+
+If this is not enough, there is more! You can pull our production worker image
+and run packit commands in there to reproduce issues with parsing packit.yaml
+or interacting with the git repo.
+```
+$ docker pull docker.io/usercont/packit-worker:prod
+```
```

Comment:
I'd drop that part, as podman/docker run pull the image automatically if necessary.